interactive_demo.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+
+import os
+
+import numpy
+
+import rclpy
+from drake_ros.systems import MovableJoints, SystemClock, TFPublisher
+from interactive_markers import InteractiveMarkerServer
+from pydrake.common.value import AbstractValue
+from pydrake.examples.manipulation_station import ManipulationStation
+from pydrake.geometry import ConnectDrakeVisualizer
+from pydrake.math import RigidTransform
+from pydrake.multibody.parsing import PackageMap
+from pydrake.multibody.plant import MultibodyPlant
```

Comment:
Ahhhh, shewt! I saw this commit about `ManipulationStation`.

Can you file a brief Drake issue? (I'll assign it to myself once you do)

Reply:
Opened https://github.com/RobotLocomotion/drake/issues/14369